include/hipSYCL/glue/generic/hiplike/clang.hpp

```diff
@@ -54,6 +54,10 @@ extern "C" unsigned __cudaPopCallConfiguration();
 
 hipError_t hipConfigureCall(dim3 gridDim, dim3 blockDim, size_t sharedMem, hipStream_t stream);
```

Comment:
This could be removed then, I guess?
And while at it, I think `__cudaPopCallConfiguration` isn't used anywhere anymore as well..?

Reply:
Done ;)